riscv-rt/CHANGELOG.md

```diff
@@ -21,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 
+- Fix v-trap core interrupt so RISCV_RT_BASE_ISA must be defined
+- Fix undefined behavior in heap initialization example documentation
```

Comment:
Why does your PR contain changes that are already merged in `master`? It is very important to keep a clean git history.